common/types/address/config.go

```diff
@@ -0,0 +1,23 @@
+package address
+
+// Config is the configuration of the address package.
+type Config struct {
+	NetworkHRP string `mapstructure:"network-hrp"`
+}
+
+var conf = &Config{
+	NetworkHRP: "sm",
+}
+
+// DefaultAddressConfig returns the default configuration of the address package.
+func DefaultAddressConfig() *Config {
+	return conf
+}
+
+// DefaultTestAddressConfig returns the default test configuration of the address package.
+func DefaultTestAddressConfig() *Config {
+	conf = &Config{
+		NetworkHRP: "smt",
```

Comment:
"smt" is too similar to "sm", this is too error-prone. per https://github.com/spacemeshos/pm/issues/106#issuecomment-1170509194, let's use "stest" for the testnet.